tests/conftest.py

```diff
@@ -1813,8 +1836,16 @@ def core_dump_and_config_check(duthosts, request):
                 core_dump_check_pass = False
 
             logger.info("Collecting running config after test on {}".format(duthost.hostname))
-            duts_data[duthost.hostname]["cur_running_config"] = \
+            # get running config after running
+            duts_data[duthost.hostname]["cur_running_config"] = {}
+            duts_data[duthost.hostname]["cur_running_config"]['host'] = \
                 json.loads(duthost.shell("sonic-cfggen -d --print-data", verbose=False)['stdout'])
+            if duthost.is_multi_asic:
+                for asic_index in range(0, duthost.facts.get('num_asic')):
+                    asic_ns = "asic{}".format(asic_index)
+                    duts_data[duthost.hostname]["cur_running_config"][asic_ns] = \
```

Comment:
Instead of using "None", could use, DEFAULT_NAMESPACE defined here:
from tests.common.helpers.constants import DEFAULT_NAMESPACE